openassessment/data.py

```diff
@@ -54,7 +54,7 @@ def _use_read_replica(queryset):
     )
 
 
-def _get_course_blocks(course_id):
```

Comment:
codecov was complaining that total project coverage was too low, so I no-covered this block (as it's expected to not be covered, since it's one of those runtime imports) so that it wouldn't complain. I then cherry-picked it onto master https://github.com/openedx/edx-ora2/pull/1843